src/gui/tray/ActivityListModel.cpp

```diff
@@ -178,11 +178,7 @@ QVariant ActivityListModel::data(const QModelIndex &index, int role) const
             }
         } else {
             // We have an activity
-            if (!a._iconData.isEmpty()) {
-                const QString svgData = "data:image/svg+xml;utf8," + a._iconData;
-                return svgData;
-            }
-            return "qrc:///client/theme/black/activity.svg";
+            return a._icon;
```

Comment:
Just for the record, activity icons are optional (as per https://github.com/nextcloud/activity/blob/master/docs/endpoint-v2.md#activity-element )
Not sure if this is now unhandled or seomthing.

Reply:
thanks,
true, the original code had the check and I forgot to keep it